sample-tests/src/test/resources/sampleweb.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://java.sun.com/xml/ns/javaee" xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd" id="WebApp_ID" version="3.0">
+     
+  <listener>
+    <listener-class>sample.customFunction.remote.MyBigdataRDFContextListener</listener-class>
```

Comment:
@beebs-systap has recently fixed the way in which web-override.xml is found. This feature works correctly in the 2.0 RC.  It should be used for these examples.